pkg/attacher/connection.go

```diff
@@ -57,43 +47,40 @@ type CSIConnection interface {
 	// should retry.
 	Detach(ctx context.Context, volumeID string, nodeID string, secrets map[string]string) (detached bool, err error)
 
-	// Probe checks that the CSI driver is ready to process requests
-	Probe(singleProbeTimeout time.Duration) error
-
 	// Close the connection
 	Close() error
```

Comment:
We can remove `Close()` as well because it can just be called from the `conn` field